drivers/entropy/Kconfig.nrf5

```diff
@@ -16,6 +16,7 @@ menuconfig ENTROPY_NRF5_RNG
 	depends on !ENTROPY_NRF_FORCE_ALT
 	depends on HAS_HW_NRF_RNG
 	select ENTROPY_HAS_DRIVER
+	select HAS_DTS_ENTROPY if !SOC_SERIES_BSIM_NRFXX
```

Comment:
What's the issue here with BSIM?

Reply:
`HAS_DTS` is not set for this target, and `HAS_DTS_ENTROPY` depends on it.

Reply:
Ok, dealing with that in #24083

Reply:
Ok, #24083 is merged, so you should be able to remove the`if !SOC_SERIES_BSIM_NRFXX`